src/components/network-chart/NetworkChartRow.js

```diff
@@ -54,6 +164,88 @@ class NetworkChartRow extends React.PureComponent<NetworkChartRowProps, State> {
     });
   };
 
+  // copy URI
+  _onDoubleClick = (_event: SyntheticEvent<>): void => {
```

Comment:
I'm not convinced that double clicking to copy is a good idea. It's not a UI practice I'm familiar with, and we already have the practice of copying the current information with a copy handler that respects the current selection, such as in the call tree.

I think this method could be removed for now, and a follow-up PR filed.

Reply:
This was one of the acceptance criteria tasks @digitarald asked for.
This is why I'd like to leave it for now and remove it in case it does not work or remove it, if it is not as important and the feature still gets accepted.

@digitarald thoughts?

Reply:
The sidebar for the stacks uses single click on the stack name; which makes this feel more like an input. Would that make sense here as well? Happy to leave it out for now if it gets hard.

Reply:
My (light) preference would be to remove it for now and move forward with this PR. It's fairly low-risk though so if it stays in I wouldn't mind too much.

Reply:
I created a follow up #1500 as this seems not scoped out and we just have a few days left before I leave.